test/ANNOTATIONS.yaml

```diff
@@ -1487,6 +1487,8 @@ memleaksfull:
     - Skip memleaks testing of DistributedMap (#19629)
   04/24/22:
     - Clean up unmanaged classes in a test (#19695)
+  09/29/22:
```

Comment:
Should this be 09/30 (the day the impact is visible, not the day of the commit)?